common/cmake/devices.cmake

```diff
@@ -304,9 +297,8 @@ function(DEFINE_DEVICE_TYPE)
     FILE ${DEFINE_DEVICE_TYPE_ARCH_XML}
     OUTPUT ${DEVICE_MERGED_FILE}
   )
-
   get_target_property_required(PYTHON3 env PYTHON3)
-  get_target_property(PYTHON3_TARGET env PYTHON3_TARGET)
+  get_target_property_required(PYTHON3_TARGET env PYTHON3_TARGET)
```

Comment:
Leave this as `get_target_property`

Reply:
OK. I am trying to get Travis CI to pass. I have moved the update_tiles script usage to be suitable for the scripts injection mechanism. CI fails with 
```
Generating arch.tiles.xml
Traceback (most recent call last):
  File "../../../../utils/update_arch_tiles.py", line 51, in <module>
    from lxml import etree as ET
ModuleNotFoundError: No module named 'lxml'
```

I though maybe that was the issue

Reply:
The problem is that I cannot reproduce the error locally

Reply:
If I had to guess your change resulted in `${PYTHON3}` being dereferenced at a location where it was not defined.  Double check

Reply:
Done